README.md

```diff
@@ -145,15 +145,26 @@ each milestone.
   the backbone models with phosphorjs widget-based views.
 * Allow widgets to be used at the top-level of the JupyterLab UI to create dashboards.
 * The model layer of the widgets will be made even more symmetric, allowing
-  widgets to be created, with traits, entirely in the front-end.  This will 
-  further support the declaritive widgets and similar work.  It will also enable
+  widgets to be created, with traits, entirely in the front-end.  This will
+  further support the declarative widgets and similar work.  It will also enable
   the mapping of typed models into the front-end to the back-end with no effort.
 
 ### Future
 
-* The backbone widgets provided with jupyter-js-widgets will be moved into their 
+* The backbone widgets provided with jupyter-js-widgets will be moved into their
   own repository or will be removed altogether.
 
+## Declarative widgets
+
+### Next
+
+* Decide where to draw the line between Web Components specific functionality and generic code that should live closer to ipywidgets / jupyter-js-widgets to prevent future breakage.
+* Migrate generic, JS modules to jupyter-js-widgets (may be a no-op).
+* Migrate accompanying Python modules to ipywidgets.
+* Graduate the declarativewidgets from the incubator.  Specifically the move jupyter-incubator/declarativewidgets code to jupyter/jupyter-web-components or another appropriately named repo.
+* Investigate with the jupyter/notebook team about upstreaming a generic bower/npm restful API.
+* Move the R code into the IRkernel and the Scala code into Gino's Scala kernel.
```

Comment:
Although I love the name you gave it, lets change this from `Gino's Scala kernel` to `Apache Toree`.